chpl/chpl-service/src/main/java/gov/healthit/chpl/attestation/domain/DeveloperAttestationSubmission.java

```diff
@@ -22,13 +24,15 @@
     private List<AttestationSubmittedResponse> responses;
     private String signature;
     private String signatureEmail;
+    private LocalDate datePublished;
 
     public DeveloperAttestationSubmission(DeveloperAttestationSubmissionEntity entity) {
         this.id = entity.getId();
         this.developer = new Developer(new DeveloperDTO(entity.getDeveloper()));
         this.period = new AttestationPeriod(entity.getPeriod());
         this.signature = entity.getSignature();
         this.signatureEmail = entity.getSignatureEmail();
+        this.datePublished = entity.getLastModifiedDate().toInstant().atZone(ZoneId.systemDefault()).toLocalDate();
```

Comment:
We have a DateUtil class somewhere in chpl-service and I wonder if that could be used here?